--- tpm2-tss-3.2.0/Makefile.am.orig	2022-02-21 21:15:40.607601135 +0100
+++ tpm2-tss-3.2.0/Makefile.am	2022-02-21 21:19:52.206238108 +0100
@@ -369,7 +369,7 @@ EXTRA_DIST += lib/tss2-tcti-libtpms.map
 if HAVE_LD_VERSION_SCRIPT
 src_tss2_tcti_libtss2_tcti_libtpms_la_LDFLAGS  = -Wl,--version-script=$(srcdir)/lib/tss2-tcti-libtpms.map
 endif # HAVE_LD_VERSION_SCRIPT
-src_tss2_tcti_libtss2_tcti_libtpms_la_LIBADD   = $(libtss2_tctildr) $(libutil)
+src_tss2_tcti_libtss2_tcti_libtpms_la_LIBADD   = $(libtss2_mu) $(libutil)
 src_tss2_tcti_libtss2_tcti_libtpms_la_SOURCES  = \
     src/tss2-tcti/tcti-common.c \
     src/tss2-tcti/tcti-libtpms.c \
