--- tpm2-tss-3.2.0/configure.ac.orig	2022-02-21 18:18:21.091907007 +0100
+++ tpm2-tss-3.2.0/configure.ac	2022-02-21 18:31:28.940972195 +0100
@@ -484,12 +484,12 @@ AS_IF([test "x$enable_integration" = "xy
 
 # Check for systemd helper tools used by make install
 AC_CHECK_PROG(systemd_sysusers, systemd-sysusers, yes)
-AM_CONDITIONAL(SYSD_SYSUSERS, test "x$systemd_sysusers" = "xyes")
+AM_CONDITIONAL(SYSD_SYSUSERS, false)
 AC_CHECK_PROG(systemd_tmpfiles, systemd-tmpfiles, yes)
-AM_CONDITIONAL(SYSD_TMPFILES, test "x$systemd_tmpfiles" = "xyes")
+AM_CONDITIONAL(SYSD_TMPFILES, false)
 
 # Check all tools used by make install
-AS_IF([test "$HOSTOS" = "Linux"],
+AS_IF([false],
     [ AC_CHECK_PROG(useradd, useradd, yes)
       AC_CHECK_PROG(groupadd, groupadd, yes)
       AC_CHECK_PROG(adduser, adduser, yes)
--- tpm2-tss-3.2.0/Makefile.am.orig	2022-02-21 18:32:44.257230839 +0100
+++ tpm2-tss-3.2.0/Makefile.am	2022-02-21 18:32:46.707217566 +0100
@@ -636,21 +636,7 @@ define make_parent_dir
 endef
 
 define make_tss_user_and_group
-    if type -p groupadd > /dev/null; then \
-        id -g tss 2>/dev/null || groupadd --system tss; \
-    else \
-        id -g tss 2>/dev/null || \
-        addgroup --system tss; \
-    fi && \
-    if type -p useradd > /dev/null; then \
-        id -u tss 2>/dev/null || \
-        useradd --system --home-dir / --shell `type -p nologin` \
-                         --no-create-home -g tss tss; \
-    else \
-        id -u tss 2>/dev/null || \
-        adduser --system --home / --shell `type -p nologin` \
-                --no-create-home --ingroup tss tss; \
-    fi
+    true
 endef
 
 define make_tss_dir
